nyc_taxi/nyc_taxi-nongeo.ipynb

```diff
@@ -172,11 +162,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scatter = hv.Scatter(df, 'tip_amount', 'fare_amount')\n",
-    "ranged = scatter.redim.range(tip_amount=(0,40), fare_amount=(0,20))\n",
-    "shaded = hd.spread(hd.datashade(ranged))\n",
-    "labelled = shaded.redim.label(tip_amount=\"Tip, $\", fare_amount=\"Fare, $\")\n",
-    "labelled"
+    "df.hvplot.scatter('tip_amount', 'fare_amount', rasterize=True, dynspread=True, cnorm='eq_hist',\n",
```

Comment:
This one is also a little difficult to see :/

<img width="825" alt="image" src="https://github.com/holoviz-topics/examples/assets/35924738/d35ea467-fb75-4947-9a83-ad074e267748">

Reply:
Looking at this plot and reading the comment, I wish it displayed a diagonal (slope with gradient 1 and y-intercept 0) to make it obvious some tips are higher than fares. I opened an issue to record this idea https://github.com/holoviz/hvplot/issues/1309.

Reply:
This is not going to be implemented soon in hvPlot so let's not hold off this PR for that.